src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -641,7 +682,71 @@ public ContainerCreation commitContainer(final String containerId,
     return request(GET, IMAGES_SEARCH_RESULT_LIST, resource,
         resource.request(APPLICATION_JSON_TYPE));
   }
+  
+  @Override
+  public void load(String image, InputStream imagePayload)
+      throws DockerException, InterruptedException {
+    load(image, imagePayload, new LoggingPullHandler("image stream"));
+  }
+  
+  @Override
+  public void load(String image, InputStream imagePayload,
+      AuthConfig authConfig, ProgressHandler handler) throws DockerException,
```

Comment:
line 694 can be aligned with the `S` in `load(String...)`. And thanks for the contribution :)